spec/valkyrie/types_spec.rb

```diff
@@ -54,10 +54,16 @@ class Resource < Valkyrie::Resource
         end.not_to output(message).to_stderr
       end
 
-      it 'equals the equivalent string if Valkyrie is configured' do
-        allow(Valkyrie.config).to receive(:id_string_equality).and_return(true)
+      context 'when String equality is configured' do
+        before { allow(Valkyrie.config).to receive(:id_string_equality).and_return(true) }
 
-        expect(resource.thumbnail_id).to eq '123'
+        it 'equals the equivalent string' do
+          expect(resource.thumbnail_id).to eq '123'
+        end
+
+        it 'is equal to the equivalent string' do
+          expect('123' == resource.thumbnail_id).to be true
```

Comment:
I think for completeness, I would also add:

```ruby
  expect(resource.thumbnail_id == '123').to be true
```

Reply:
that's the test on https://github.com/samvera/valkyrie/pull/821/files/d9f0895504c17dcd3056e783a24d7bab12742642#diff-b8273acc787c804ca17fe99782a00941R61, right?

this version needs to embed the equality check in the expectation to satisfy `rubocop-rspec`, which gets mad if you put a literal value in `expect`'s actual. `expect('123').to eq resource.thumbnail_id` is equivalent, but fails style check (: